docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_9.md

```diff
@@ -0,0 +1,86 @@
+
+As part of the NCPI Effort, methods have been developed to allow users to bring datasets from the other NIH platforms into CAVATICA for combined research projects. One external dataset that is supported is the [Genotype Tissue Expression (GTEx) Program](https://commonfund.nih.gov/gtex). GTEx was funded to study the relationship between genetic variants (inherited changes in DNA sequence) and gene expression (how genes are turned on and off) in multiple human tissues and across individuals. Their datasets can serve as great controls for RNA-Seq experiments, comparing expression in GTEx's "normal" brain tissue to Kids First's brain cancer tissue.
+
+The previous example provides a walkthrough for identifying RNA-Seq datasets from the [Kids First Data Resource Portal](https://portal.kidsfirstdrc.org/) and pushing them to [CAVATICA](https://cavatica.sbgenomics.com/) for analysis. This supplemental lesson will demonstrate how to find GTEx datasets on the [NHGRI Analysis Visualization and Informatics Lab-space (AnVIL)](https://anvilproject.org/) Portal] and import these files into a CAVATICA project for a combined analysis with Kids First data.
+
+
+## Step 1: Identify GTEx Files on AnVIL and Export a PFB
+- Navigate to [the AnVIL Portal](https://gen3.theanvil.io/login) and login using NIH or Google credentials.
+    - <img src="./rna-seq-images/rna-seq-9-01.png" width="400"/>
+- Using the [AnVIL Data Explorer](https://gen3.theanvil.io/explorer), identify the GTEx files for your analysis. For more information on using AnVIL Gen3, see [their support documentation here](https://anvilproject.org/learn/introduction/getting-started-with-gen3).
+    - <img src="./rna-seq-images/rna-seq-9-02.png" width="400"/>
+- Once you have identified the subset of files for your analysis, return to the top of the screen and choose `Export All to PFB`. This will create a manifest of files on the AnVIL Portal. This process may take several minutes, depending upon the number of files for export.
+    - <img src="./rna-seq-images/rna-seq-9-03.png" width="400"/>
+- Once the PFB is created, download it to your local machine. This downloaded PFB file contains the information CAVATICA needs to pull these files into your CAVATICA project.
+    - <img src="./rna-seq-images/rna-seq-9-04.png" width="400"/>
+- Before navigating away from the AnVIL Portal, you also must generate an API key. This will be used to authenticate your access to the files in the PFB and allow the transfer of GTEx data into CAVATICA. Select your username in the top right of the AnVIL Portal. Select `Create API Key` in the top left of the page that opens. After a key is generated, download this key by choosing `Dowload json`. A file will save to your local machine.
+    - <img src="./rna-seq-images/rna-seq-9-06.png" width="400"/>
```

Comment:
This image is a screenshot of the Cavatica home page, but you haven't told the reader how to get there. I think this image would be more appropriate in the next section. If you do include an image here, I think it should be a screenshot of the "Create API key".

Reply:
corrected image reference on line 28 from a Cavatica login screen on rna-seq-9-06.png to an AnVIL Create API Key